src/librustc_parse/lexer/mod.rs

```diff
@@ -471,16 +471,9 @@ impl<'a> StringReader<'a> {
 
     /// As symbol_from, with the text normalized into Unicode NFC form.
     fn nfc_symbol_from(&self, start: BytePos) -> Symbol {
```

Comment:
This function is tiny and used only once, it can be inlined.

Reply:
I separated it out for consistency with `symbol_from`, `symbol_from_to` etc. If you wish i can inline it back.

Reply:
I think inlining would be better here.

Reply:
Done.